tests/Stubs/Models/Test.php

```diff
@@ -6,4 +6,5 @@
 
 class Test extends Model
 {
+    public $timestamps = false;
```

Comment:
why do we need this? I think I might be missing something 🙈

Reply:
Because the Test model is persisted to the database for the tests I either had to add `$table->timestamps();` to the migration or just remove the timestamps for the model. I decided to go for the last one as the timestamps are irrelevant for the test anyway 😉